concourse/pipelines/templates/jobs/test-multinode-tpl.yml

```
@@ -35,7 +35,7 @@
       trigger: true
 {% if x.with_upgrade == 'true' %}
     - get: pxf_artifact
-      resource: pxf5-gp6-el7-released   # for upgrade test
+      resource: pxf6-gp6-el8-released
```

keep the comment here
```suggestion
      resource: pxf6-gp6-el8-released # for upgrade test
```